### PR TITLE
Adds ssh tunnel methods to utils/ssh.py

### PR DESCRIPTION
**Pull request by blrm**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit f6de997c9e056ebce60a201d949bce5a60923bd2:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: utils/ssh.py [hunk 1]
```diff
@@ -119,10 +122,6 @@ def connect(self, hostname=None, **kwargs):
             # Only install ssh keys if they aren't installed (or currently being installed)
             return super(SSHClient, self).connect(**self._connect_kwargs)
 
-    def open_sftp(self, *args, **kwargs):
```
Comment (psav) [inline utils/ssh.py]: Why does this get removed?

File: utils/ssh.py [hunk 2]
```diff
@@ -163,11 +162,6 @@ def run_command(self, command, timeout=RUNCMD_TIMEOUT):
             return SSHResult(exit_status, output)
         except paramiko.SSHException as exc:
             logger.exception(exc)
-        except socket.timeout as e:
```
Comment (psav) [inline utils/ssh.py]: Is there a reason to remove this logging?

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by psav** [comment]
@blrm is this functionality in your branch? What do we need to do to this one?

File: utils/ssh.py (hunk 1 above)
Comment (mfalesni) [inline utils/ssh.py]: Yeah, this should not get removed. I added that to `.connect()` inside. Doing this will nuke some tests I believe.

Comment (psav) [inline utils/ssh.py]: In light of that, what's the future of this PR?

**Comment by jkrocil** [comment]
@blrm Do we still want this? If yes, let's move it forward; otherwise, can we close this PR?

**Comment by RonnyPfannschmidt** [comment]
btw, i think this breaks usage against containers

**Pull request closed without merging**